templates/shared_services/nexus-cert/scripts/letsencrypt.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+set -e
+
+script_dir=$(realpath $(dirname "${BASH_SOURCE[0]}"))
+if [[ -z ${STORAGE_ACCOUNT} ]]; then
+  echo "STORAGE_ACCOUNT not set"
+  exit 1
+fi
+
+echo "Checking for index.html file in storage account"
+
+# Create the default index.html page
+cat << EOF > index.html
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml"><head><meta charset="utf-8"/><title></title></head><body></body></html>
+EOF
+
+indexExists=$(az storage blob list -o json \
+    --account-name "${STORAGE_ACCOUNT}" \
+    --auth-mode login \
+    --container-name '$web' \
+    --query "[?name=='index.html'].name" \
+    | jq 'length')
+
+if [[ ${indexExists} -lt 1 ]]; then
+    echo "Uploading index.html file"
+
+    az storage blob upload \
+        --account-name "${STORAGE_ACCOUNT}" \
+        --auth-mode login \
+        --container-name '$web' \
+        --file index.html \
+        --name index.html \
+        --no-progress \
+        --only-show-errors
+
+    # Wait a bit for the App Gateway health probe to notice
+    echo "Waiting 30s for health probe"
+    sleep 30s
+else
+    echo "index.html already present"
+fi
+
+ledir=$(cd ${script_dir} && cd ../ && pwd)/letsencrypt
```

Comment:
nit: could this be simplified?

```suggestion
ledir="${script_dir}/../letsencrypt"
```